test/addons/olm/fetch

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
```

Comment:
This commit fixes #1337 - please add `Fixes: #1337` to the commit message footer, so the issue will be closed when we merge this.